examples/message-ubench/main.pony

```diff
@@ -113,8 +115,12 @@ actor SyncLeader
   var _total_count: U64 = 0
   var _current_t: I64 = 0
   var _last_t: I64 = 0
+  var _run_done: Bool = false
+  var _timers: Timers tag
+  var _interval_timer: Timer tag
```

Comment:
FYI Wink.

`Timers` and `Timer` are both actors so don't need the `tag`.

`Timer` rather than `Timer tag` is fine and normal practice.

Reply:
fixed